draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1616,17 +1616,17 @@ stream, tag = Encrypt(ZeroPad({}, len), {}, key, nonce)
 return stream
 ~~~
 
-This is equivalent to encrypting a `len` all-zero bits message without associated data, and discarding the authentication tag.
+This is equivalent to encrypting a `len` all-zero bits message without associated data and discarding the authentication tag.
 
-Instead of relying on the generic `Encrypt` function, implementations can skip the finalization step.
+Instead of relying on the generic `Encrypt` function, implementations can omit the `Finalize` function.
 
 After initialization, the `Update` function is called with constant parameters, allowing further optimizations.
 
 # AEGIS as a Message Authentication Code
 
 All AEGIS variants can be used to construct a MAC.
```

Comment:
I would think defining MAC in brackets is required by the style guides, although I haven't checked this, and it would be annoying given the section title. We could leave it and see what the editor/whoever does.

Reply:
We didn't define what an AEAD is either, but given the intended audience, this is probably fine.

Reply:
Given the audience, I still don't think this is useful. The only important thing to highlight is that it is not a keyed hash function, but we already mention this.

Reply:
`MAC` was replaced with `Message Authentication Code (MAC)`.

I originally thought you actually wanted to define what that class of function is.